app/controllers/car_controller.rb

```diff
@@ -0,0 +1,29 @@
+class CarController < ApplicationController
+
+  def index
+    @cars = Cars.all
+  end
+
+  def create
+    car = Car.epa_vehicle_by_id(id: params[:epa_id])
+    new_car = Car.import_car_from_epa(car)
+    if new_car.save
+      render 'show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.